### DISCUSSION TITLE
自动选择节点这个功能是真的垃圾

### DISCUSSION THREAD

**Discussion by TAEzio** [getlantern/lantern-forum-cn#958] [Ideas]
一分钟能换几十次节点，一换节点就连不上，每次都是看运气，每次连接都得等上一会，连上个能用的节点才有网

**Comment by Xisong-Feng** [comment]
真的太出生了，每次刷节点都感觉像玄学一样，实在是太影响使用了

**Comment by mofasioa** [comment]
一年不如一年 以前都是固定节点 现在1-2分钟就变 而且一会连上一会断开
就不能弄个自己选择的节点吗?

**Comment by MaxMKMax** [comment]
@TAEzio @Francis-ui-cmd @mofasioa 

您好，谢谢您的反馈。蓝灯已经在正视这个问题了。 我们正在修复问题和进行优化。请您耐心等待。非常抱歉影响了您的体验感。谢谢您的等待与支持。

您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

go to Support (转到支持)
select Report an Issue (选择报告问题)
choose issue type (选择问题类型)
include additional details (包括额外的详细信息)
hit Send Report (点击发送报告)

当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队. 不好意思, 需要您的帮忙. 谢谢!

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。

**Reply by TAEzio** [reply]
78084038

**Reply by MaxMKMax** [reply]
@TAEzio 谢谢您啦!
